packages/session-replay-browser/src/session-replay.ts

```diff
@@ -205,9 +205,17 @@ export class SessionReplay implements AmplitudeSessionReplay {
       if (this.config.debugMode) {
         eventProperties[SESSION_REPLAY_DEBUG_PROPERTY] = this.getSessionReplayDebugPropertyValue();
       }
+      void this.addCustomRRWebEvent('get-sr-props', {
+        shouldRecord,
+        eventProperties: eventProperties,
+      });
       return eventProperties;
     }
 
+    void this.addCustomRRWebEvent('get-sr-props', {
```

Comment:
I think we should probably enum our custom event types `'get-sr-props'`. Creating our own mini taxonomy here :)